x-pack/legacy/plugins/siem/public/pages/detection_engine/translations.ts

```diff
@@ -86,3 +86,17 @@ export const USER_UNAUTHENTICATED_MSG_BODY = i18n.translate(
       'You do not have the required permissions for viewing the detection engine. For more help, contact your administrator.',
   }
 );
+
+export const ML_RULES_DISABLED_MESSAGE = i18n.translate(
+  'xpack.siem.detectionEngine.mlRulesDisabledMessageTitle',
+  {
+    defaultMessage: 'ML rules require Platinum License & ML Admin Permissions',
+  }
+);
+
+export const ML_RULES_UNAVAILABLE = (totalRules: number) =>
+  i18n.translate('xpack.siem.detectionEngine.mlUnavailableTitle', {
+    values: { totalRules },
+    defaultMessage:
+      '{totalRules} {totalRules, plural, =1 {rule} other {rules}} require Machine Learning to enable.',
```

Comment:
`require` should be `requires` if it is plural vs singular.

Reply:
🎉 
<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/78413685-cf064580-75d5-11ea-90fc-45894e95e7b9.png" />
  <img width="300" src="https://user-images.githubusercontent.com/2946766/78413691-d88fad80-75d5-11ea-89aa-4b3766aafa66.png" />
</p>